src/main/java/it/gov/pagopa/reward/utils/MongoRequestRateTooLargeUtilities.java

```diff
@@ -0,0 +1,23 @@
+package it.gov.pagopa.reward.utils;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MongoRequestRateTooLargeUtilities {
+
+  @SuppressWarnings("squid:S1444")
+  public static int maxRetry;
+  @SuppressWarnings("squid:S1444")
+  public static int maxMillisElapsed;
+
+  @SuppressWarnings("squid:S1118")
```

Comment:
Let's remove this warning, just rename the class into
MongoRequestRateTooLargeConfig